docs/gui/biomes.rst

```diff
@@ -0,0 +1,17 @@
+gui/biomes
+==========
+
+.. dfhack-tool::
+    :summary: Visualize and inspect biome regions on the map.
+    :tags: fort inspection map
+
+This script shows the boundaries of the biome regions on the map.
+Hover over a biome entry in the list to get detailed info about it.
```

Comment:
How about also allowing the player to hover the mouse cursor over the map to select the region in the list? If the keyboard cursors are used and the mouse hasn't moved, then that can take precedence to move the list cursor. Then when the mouse moves, it can be the selector again.

Reply:
implemented... Now that I re-read your comment, probably not exactly what you had in mind, but hopefully close enough